### PR TITLE
Doc 10892a

### PR DESCRIPTION
**Pull request by tonyjhillman**

### CODE REVIEW

**Review by hyunjuV — COMMENTED** [review]
Lines 55 and 56 also need to have 7.1.3 changes to 7.2.0:wget https://packages.couchbase.com/releases/7.1.3/couchbase-server-tools_7.1.3-linux_x86_64.tar.gz
tar -xf couchbase-server-tools_7.1.3-linux_x86_64.tar.gz

The rest look good.

**Comment by tonyjhillman** [comment]
Thank you for catching that! I will fix and merge.

File: modules/cli/pages/cli-intro.adoc [hunk 1]
```diff
@@ -34,21 +34,26 @@ You can collect diagnostics through the command-line interface by using either t
...
-* https://packages.couchbase.com/releases/7.1.3/couchbase-server-tools_7.1.3-linux_aarch64.tar.gz[https://packages.couchbase.com/releases/7.1.3/couchbase-server-tools_7.1.3-linux_aarch64.tar.gz]
-* https://packages.couchbase.com/releases/7.1.3/couchbase-server-tools_7.1.3-macos_x86_64.zip[https://packages.couchbase.com/releases/7.1.3/couchbase-server-tools_7.1.3-macos_x86_64.zip]
-* https://packages.couchbase.com/releases/7.1.3/couchbase-server-tools_7.1.3-windows_amd64.zip[https://packages.couchbase.com/releases/7.1.3/couchbase-server-tools_7.1.3-windows_amd64.zip]
+
+* https://packages.couchbase.com/releases/7.2.0/couchbase-server-tools_7.2.0-linux_x86_64.tar.gz[https://packages.couchbase.com/releases/7.2.0/couchbase-server-tools_7.2.0-linux_x86_64.tar.gz]
+
+* https://packages.couchbase.com/releases/7.2.0/couchbase-server-tools_7.2.0-linux_aarch64.tar.gz[https://packages.couchbase.com/releases/7.2.0/couchbase-server-tools_7.2.0-linux_aarch64.tar.gz]
+
+* https://packages.couchbase.com/releases/7.2.0/couchbase-server-tools_7.2.0-macos_x86_64.zip[https://packages.couchbase.com/releases/7.2.0/couchbase-server-tools_7.2.0-macos_x86_64.zip]
+
```
Comment (hyunjuV) [inline modules/cli/pages/cli-intro.adoc:47]: There is now a macos_arm64 build -- so need to add this as well:
https://packages.couchbase.com/releases/7.2.0/couchbase-server-tools_7.2.0-macos_arm64.zip

File: modules/cli/pages/cli-intro.adoc [hunk 2]
```diff
@@ -63,3 +68,14 @@ If your Apple Silicon Mac does not already have Rosetta installed, it can be ins
 ----
 softwareupdate --install-rosetta
 ----
```
Comment (hyunjuV) [inline modules/cli/pages/cli-intro.adoc:70]: Lines 64 through 70 are no longer needed since there is a macos arm64 package now.

Comment (tonyjhillman) [reply modules/cli/pages/cli-intro.adoc:70]: Okay, I've removed.

File: modules/cli/pages/cli-intro.adoc (hunk 1 above)
Comment (tonyjhillman) [reply modules/cli/pages/cli-intro.adoc:47]: I've added.

**Pull request merged**